specs/opcode/02MUL_04DIV_06MOD.md

```diff
@@ -73,4 +105,4 @@ It always computes `y = (a * x) % 2**256`, annotate stack as \[a, x, ...\] and \
 
 ## Code
 
-See `src/zkevm_specs/opcode/mul.py`
+See `src/zkevm_specs/evm/execution/mul.py`
```

Comment:
Shouldn't we call it `mul_div_mod.py` since it covers the 3 cases?

Reply:
shall we also rename `add.py` into `add_sub.py`? Further, in the rust circuit, we also use `add.rs` and `mul.rs` instead of `add_sub.rs` etc. Do you think we should change it correspondingly for consistency?

Reply:
@han0110 @ChihChengLiang @ed255 I'll leave the file naming decision to you guys.

Reply:
I like `mul_div_mod.py` and `add_sub.rs` because you can search `sub` or `div` in the editor, and it finds the right file for devs to look into. Changing all of them for consistency sounds good.

Reply:
@ChihChengLiang renamed. could you help review the pr?